### PR TITLE
fix(components): [tabs] (#10519)

### PR DESCRIPTION
**Pull request by ac51300696**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/tabs/src/tabs.tsx [hunk 1]
```diff
@@ -119,16 +119,16 @@ export default defineComponent({
...
 
-    const handleTabClick = (
+    const handleTabClick = async (
       tab: TabsPaneContext,
       tabName: TabPaneName,
       event: Event
     ) => {
       if (tab.props.disabled) return
-      setCurrentName(tabName)
+      await setCurrentName(tabName)
```
Comment (tolking) [inline packages/components/tabs/src/tabs.tsx:131]: `update:modelValue` -> `tabChange` -> `tabClick`

Do you think this sequence of events makes more sense?

Comment (ac51300696) [reply packages/components/tabs/src/tabs.tsx:131]: https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuaW1wb3J0IHR5cGUgeyBUYWJzUGFuZUNvbnRleHQgfSBmcm9tICdlbGVtZW50LXBsdXMnXG4gIGNvbnN0IGFjdGl2ZU5hbWUgPSByZWYoJ2ZpcnN0JylcblxuY29uc3QgaGFuZGxlQ2xpY2sgPSAodGFiOiBUYWJzUGFuZUNvbnRleHQsIGV2ZW50OiBFdmVudCkgPT4ge1xuICBtc2cudmFsdWUgPSBgYWN0aXZlTmFtZTogJHthY3RpdmVOYW1lLnZhbHVlfSwgY3VycnRhYjogJHt0YWIucGFuZU5hbWV9YFxufVxuXG5jb25zdCBtc2cgPSByZWYoJycpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtdGFicyB2LW1vZGVsPVwiYWN0aXZlTmFtZVwiIGNsYXNzPVwiZGVtby10YWJzXCIgQHRhYi1jbGljaz1cImhhbmRsZUNsaWNrXCI+XG4gICAgPGVsLXRhYi1wYW5lIGxhYmVsPVwiVXNlclwiIG5hbWU9XCJmaXJzdFwiPlVzZXI8L2VsLXRhYi1wYW5lPlxuICAgIDxlbC10YWItcGFuZSBsYWJlbD1cIkNvbmZpZ1wiIG5hbWU9XCJzZWNvbmRcIj5Db25maWc8L2VsLXRhYi1wYW5lPlxuICAgIDxlbC10YWItcGFuZSBsYWJlbD1cIlJvbGVcIiBuYW1lPVwidGhpcmRcIj5Sb2xlPC9lbC10YWItcGFuZT5cbiAgICA8ZWwtdGFiLXBhbmUgbGFiZWw9XCJUYXNrXCIgbmFtZT1cImZvdXJ0aFwiIGRpc2FibGVkPlRhc2s8L2VsLXRhYi1wYW5lPlxuICA8L2VsLXRhYnM+XG4gIDxkaXY+XG4gICAge3ttc2d9fVxuICA8L2Rpdj5cbiAgPGRpdj5cbiAgICB7e21zZ0NoYW5nZX19XG4gIDwvZGl2PlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHt9XG59IiwiX28iOnt9fQ==

Comment (ac51300696) [reply packages/components/tabs/src/tabs.tsx:131]: activeName always equie currtab name

Comment (ac51300696) [reply packages/components/tabs/src/tabs.tsx:131]: maybe I'm too used to the past。 in may case，i can also get currTabName not by activeTabName。

Comment (tolking) [reply packages/components/tabs/src/tabs.tsx:131]: Thank for your contribution

IMO, There is no problem with the current event triggering sequence. You need to pay attention to the rational use of tabClick events.

**Pull request closed without merging**